pkgs/applications/virtualization/imgcrypt/default.nix

```diff
@@ -2,20 +2,21 @@
 
 buildGoModule rec {
   pname = "imgcrypt";
-  version = "1.1.7";
+  version = "1.1.8";
 
   src = fetchFromGitHub {
     owner = "containerd";
     repo = pname;
     rev = "v${version}";
-    sha256 = "sha256-VGP63tGyYD/AtjEZD1uo8A2I/4Di7bfLeeaNat+coI4=";
+    hash = "sha256-FLqFzEEfgsKcjAevhF6+8mR3zOUjfXyfWwWsxVOcdJU=";
   };
 
+  vendorHash = null;
```

Comment:
This does not look right?

Reply:
The dependencies are vendored in the source repo. So it's fine to be `null`.